src/core/remote.js

```diff
@@ -1660,23 +1646,21 @@ Remote.prototype.requestBookOffers = function(options_, callback_) {
 Remote.prototype.requestWalletAccounts = function(options_, callback_) {
   utils.assert(this.trusted); // Don't send secrets.
 
-  let options;
-  const callback = callback_;
+  const options = {callback: callback_};
 
   if (lodash.isPlainObject(options_)) {
-    options = lodash.merge({}, options_);
+    lodash.merge(options, options_);
   } else {
-    log.warn('DEPRECATED: First argument to request constructor should be'
-      + ' an object containing request properties');
-
-    options = {
-      seed: arguments[0]
-    };
+    lodash.merge(options, makeOptions(
+      'wallet_accounts',
+      ['seed'],
+      Array.prototype.slice.call(arguments)
```

Comment:
Crossed my mind when I was writing this, thought lodash had a slice thing. I'll use that instead